public/v2/apps/phpldapadmin.json

```diff
@@ -0,0 +1,77 @@
+{
+    "captainVersion": "2",
+    "documentation": "https://github.com/osixia/docker-openldap",
+    "description": "OpenLDAP with administration interface",
+    "displayName": "OpenLDAP + phpLDAPadmin",
+    "dockerCompose": {
+        "version": "3.8",
+        "services": {
+            "$$cap_appname": {
+                "image": "osixia/openldap:$$cap_openldap_version",
+                "restart": "always",
+                "ports": ["389:389", "636:636"],
```

Comment:
Do ports really need to be exposed? If it's for internal use, you can access this by `srv-captain--$$cap_appname:389` without port mapping.

Reply:
In my case, I needed it to be accessible from other machines. I can remove it and explain how to activate it in the end instructions.

Reply:
That'd be perfect! Thanks!